api/lib_test.go

```diff
@@ -175,6 +175,52 @@ func TestHandle(t *testing.T) {
 	assert.Equal(t, send.Amount, balance)
 }
 
+func TestMigrate(t *testing.T) {
```

Comment:
@alpe This is a valid go test that this is working. You can use this as a template for the wasmd tests.

Note that you need to update the test contracts in wasmd as well when you update to this. Just copy them from this repo, I built them from cosmwasm:0.9 via rust-optimizer